src/object.c

```diff
@@ -733,10 +733,12 @@ size_t objectComputeSize(robj *o, size_t sample_size) {
         if (o->encoding == OBJ_ENCODING_HT) {
             d = o->ptr;
             di = dictGetIterator(d);
-            asize = sizeof(*o)+sizeof(dict)+(sizeof(struct dictEntry*)*dictSlots(d));
+            int dicthtNum = dictIsRehashing(d) + 1;
```

Comment:
I did some tests and it turns out you are right.
Thanks for your kindly reminding. 
Fixed that.